### PR TITLE
Version 48

### PR DESCRIPTION
**Pull request by tarngerine**

### CODE REVIEW

**Comment by tarngerine** [comment]
just putting this up for a diff view :p

File: Headers/MSContentDrawView.h [hunk 1]
```diff
@@ -94,15 +95,17 @@
 - (void)refreshSidebarWithMask:(unsigned long long)arg1;
 - (id)pages;
 - (id)currentPage;
-- (id)currentView;
+- (id)contentDrawView;
```
Comment (tarngerine) [inline Headers/MSContentDrawView.h]: sigh...

File: Headers/MSTextLayer.h [hunk 2]
```diff
@@ -98,8 +98,6 @@
 - (BOOL)compareAttributes:(id)arg1 withAttributes:(id)arg2;
 - (void)syncTextStyleAttributes;
 - (id)sharedObject;
-- (id)baselineOffsets;
```
Comment (tarngerine) [inline Headers/MSTextLayer.h]: ugh how do i get this now

File: Headers/MSImmutableTextLayer.h [hunk 3]
```diff
@@ -10,36 +10,29 @@
...
 - (double)baselineAdjustmentForLayoutManager:(id)arg1;
 - (id)createTextStorage;
 - (id)keysDifferingFromObject:(id)arg1;
 - (BOOL)isEqualForDiffToObject:(id)arg1;
 - (BOOL)hasDefaultValues;
-@property(readonly, nonatomic) double firstBaselineOffset;
-@property(readonly, nonatomic) double firstLineCapOffset;
-@property(readonly, nonatomic) struct CGRect lineFragmentBounds;
-@property(readonly, copy, nonatomic) NSArray *baselineOffsets;
+@property(readonly, nonatomic) MSTextLayout *textLayout; // @synthesize textLayout=_textLayout;
```
Comment (tarngerine) [inline Headers/MSImmutableTextLayer.h]: how would one get an MSImmutableTextLayer from an MSTextLayer to access this

**Comment by abynim** [comment]
@tarngerine for now this should probably go to the `betas` branch instead of `master`.

**Comment by exevil** [comment]
@tarngerine: Merged your PR manually, thanks.

**Pull request closed without merging**